.prettierignore

```diff
@@ -0,0 +1,11 @@
+.circleci/
+build-configs/
```

Comment:
I think we can use prettier for build configs, docs, release notes, tools and translations as well. Or did you have a reason to exclude it?

Reply:
We have no valid point to check them in the pipeline, we do only check web and check native. Therefore i did not add. If i just overlook it, we can add it. But lets add it to a new issue. I would love to have this merged asap.